overlord/devicestate/devicestate_install_mode_test.go

```diff
@@ -3662,3 +3663,138 @@ func (s *deviceMgrInstallModeSuite) TestEncryptionSupportInfoWithFdeHook(c *C) {
 		c.Check(res, DeepEquals, tc.expected, Commentf("%v", tc))
 	}
 }
+
+type installStepSuite struct {
+	deviceMgrSystemsBaseSuite
+}
+
+var _ = Suite(&installStepSuite{})
+
+func (s *installStepSuite) SetUpTest(c *C) {
+	classic := false
```

Comment:
Shouldn't this be eventually true? Maybe add a TODO if we need a change on how things are set-up for the test?

Reply:
Nice catch and indeed, it should be classic. I fixed that now.